src/spatialdata/_io/io_raster.py

```diff
@@ -130,9 +134,12 @@ def _get_group_for_writing_transformations() -> zarr.Group:
             return group.require_group(name)
         return group["labels"][name]
 
-    # convert channel names to channel metadata
+    # convert channel names to channel metadata in omero
     if raster_type == "image":
-        group_data.attrs["channels_metadata"] = fmt.channels_to_metadata(raster_data, channels_metadata)
+        metadata["metadata"] = {"omero": {"channels": []}}
```

Comment:
so "technically" here I should be checking that metadata doesn't already have this key, and also with respect to the omero dict, yet the only time that the `_write_raster` is exposed, via [`_write_image`](https://github.com/scverse/spatialdata/blob/18b32e76d8b87600b15f5253e5e3b582f8d7bf1f/src/spatialdata/_core/spatialdata.py#L1042), it is not allowed for the metadata to be passed. What would you suggest? I don't really see a real use case, so I would suggest to keep it as is and use them effectively as kwargs as it is now.

Reply:
I agree, I also would keep this as it is.